doc/api/test.md

```diff
@@ -1265,6 +1273,9 @@ changes:
   * `forceExit`: {boolean} Configures the test runner to exit the process once
     all known tests have finished executing even if the event loop would
     otherwise remain active. **Default:** `false`.
+  * `globPatterns`: {Array} An array containing the list of glob patterns to
```

Comment:
why is this needed? why not pass `run({ files: globSync('**/*.test.js') })`

Reply:
It's needed because we track changed files in watch mode: https://github.com/nodejs/node/blob/0b1ff6965e3bb3cedfe3563f4ddefc33e7fa69d2/lib/internal/test_runner/runner.js#L423.

https://github.com/nodejs/node/pull/53114 regressed heavily for me with the use of `run({ watch: true })`. The alternative to adding this option is to revert https://github.com/nodejs/node/pull/53114, which I though was less preferable.

Reply:
good point. So I would just add validation & make `files` and `globPatterns` mutual exclusive and document that